### DISCUSSION TITLE
Donations

### DISCUSSION THREAD

**Discussion by iorbita** [templaza/astroid-framework#363] [Ideas]
Hi,
just because I appreciate the work done by Templaza team and its contributors, why not insert a donation button on the [framework page](https://astroidframe.work/)? I will be the first to contribute...

Lorenzo

**Comment by Chacapamac** [comment]
Sure, I will go with something a bit discreet. Not to rich here, but I will donate.

**Comment by iorbita** [comment]
...I tried to make a donation but my card is not accepted, I got this type of message "Your card does not support this type of purchase", strange this is the first time it happens, I buy everything on the web with this card ...

**Reply by sonvnn** [reply]
I will recheck the donate button after complete Astroid 3. Thank you very much :)

**Comment by iorbita** [comment]
...any news on this issue?

**Reply by sonvnn** [reply]
I switch donate button from Stripe to Paypal. May you help me try it again via https://www.paypal.com/paypalme/sonvnn ?

Thanks & Best Regards,
Sonny

**Reply by iorbita** [reply]
... done ;)

**Reply by sonvnn** [reply]
Thank you very much :)

**Comment by Chacapamac** [comment]
I just test the system, seem to work perfectly. I will send some cash to my Paypal and send it.

**Only two questions here:** 
• The text accompanying the donation button —> “TEMPLAZA MEDIA TECHNOLOGY JOINT STOCK COMPANY” is kind of cryptic for me (I’m french) —> What about something more to the point —> TEMPLAZA MEDIA TECHNOLOGY — DONATE ?????

• Is this button on Astroid Website?

**Reply by sonvnn** [reply]
Hi,

This is paypal account of my company in Vietnam. TemPlaza Media Technology JSC is our business name which is registered in my country.

**Reply by sonvnn** [reply]
I tried to join Sponsored program of Github. Unfortulately, it is not support for my country yet.

**Reply by Chacapamac** [reply]
**Cool — Do you have the donate button on Astroid WebSite?**

**Just an advice for Astroid WebSite** — You shoild replace the main menu item “Partners” by “Templates” — This will generate more link to all partners Templates business.

**Reply by sonvnn** [reply]
@Chacapamac Thank you. I will do it when I complete Two template with Astroid 3

**Reply by Chacapamac** [reply]
Your Welcome, Good Day!
